core/src/test/scala/quasar/physical/blobstore/BlobstoreDatasourceSpec.scala

```diff
@@ -48,25 +45,25 @@ abstract class BlobstoreDatasourceSpec[F[_]: Effect] extends EffectfulQSpec[F] {
 
   "prefixedChildPaths" >> {
 
-    "list nested children" >>* {
+    "list nested children" in IO {
```

Comment:
You don't need to wrap all the examples in `IO`, `cats-effect-testing` adds support for treating `IO` values as results (automatically running them with a timeout), it doesn't mandate that all examples be in `IO`. I would have expected all the examples to compile as-is, once you replaced `>>*` with `>>`.